docs/loadflow/loadflow.md

```diff
@@ -16,10 +20,14 @@ equal to zero and $1$. In case of a branch with voltage or phase control, the $\
 
 ![Pi model](pi-model.svg)
 
+### HVDC line
+
+A specific type of branch relying on a specific type of model is the HVDC line (High Voltage Direct Current). This line is connected to the rest of the AC network through HVDC converter stations, that can be either LCC (Line-Commutated Converter) or VSC (Voltage-source Converter).
```

Comment:
"A specific type of branch relying on a specific type of model is the HVDC line": are you sure this sentence is useful ?
And maybe Voltage-**S**ource Converter